routes/api-router.js

```diff
@@ -1,7 +1,7 @@
 const express = require('express');
 const { signIn, signUp } = require('../controllers/auth-controller');
 const { getFeed, registerFeed, deleteFeed } = require('../controllers/feeds-controller');
-const authCheck = require('../middlewares/auth-check');
+const authCheck = require('../middlewares/check-auth');

```

Comment:
change ('../middlewares/auth-check') to ('../middlewares/check-auth') because it's my homework